tests/models/reading_comprehension/bidirectional_attention_test.py

```diff
@@ -49,3 +50,13 @@ def test_trains_and_loads_correctly(self):
         # TODO(matt): fix the randomness that occurs here.
         # assert_allclose(model.model.predict(validation_input),
         #                 loaded_model.model.predict(validation_input))
+
+    def test_get_best_span(self):
+        # Note that the best span cannot be (1, 1) since even though
```

Comment:
Note that with our current training regime, this isn't actually correct.  See here: https://github.com/allenai/deep_qa/blob/master/tests/data/instances/character_span_instance_test.py#L62.  We need to change the end index to be exclusive, not inclusive.  But, so you can use the model that you've been training, you should probably change this code to assume inclusive span_end, and we can make a separate PR later to change it, and retrain the model.